pkg/flowaggregator/flowaggregator.go

```diff
@@ -252,17 +253,29 @@ func (fa *flowAggregator) initExportingProcess() error {
 		return fmt.Errorf("got error when initializing IPFIX exporting process: %v", err)
 	}
 	fa.exportingProcess = ep
-	fa.templateID = fa.exportingProcess.NewTemplateID()
-	templateSet := ipfix.NewSet(ipfixentities.Template, fa.templateID, false)
 
-	bytesSent, err := fa.sendTemplateSet(templateSet)
+	fa.templateIDv4 = fa.exportingProcess.NewTemplateID()
+	templateSet := ipfix.NewSet(ipfixentities.Template, fa.templateIDv4, false)
+	bytesSent, err := fa.sendTemplateSet(templateSet, false)
 	if err != nil {
 		fa.exportingProcess.CloseConnToCollector()
 		fa.exportingProcess = nil
-		fa.templateID = 0
+		fa.templateIDv4 = 0
 		return fmt.Errorf("sending template set failed, err: %v", err)
```

Comment:
may specify which template set in the error message (IPv4 / IPv6), same comment below